test/components/FeatureFlagRenderer.test.js

```diff
@@ -173,11 +176,13 @@ describe("components/FeatureFlagRenderer", () => {
       });
 
       it("does not listen to onReady", () => {
-        expect(ldClientWrapperOnReady).not.toHaveBeenCalled();
+        expect(jest.fn((callback) => {
+          callback();
+        })).not.toHaveBeenCalled();
       });
 
       it("does not listen to change event", () => {
-        expect(ldClientWrapperOn).not.toHaveBeenCalled();
+        expect(jest.fn()).not.toHaveBeenCalled();
```

Comment:
@georgeciubotaru  All of the test changes here can be avoided by adding a second arg of `{ disableLifecycleMethods: true }` to the `shallow` method here: https://github.com/TrueCar/react-launch-darkly/blob/master/test/components/FeatureFlagRenderer.test.js#L24

It seems that with enzyme 3, they changed the default behavior of life cycle methods for shallow renders: https://airbnb.io/enzyme/docs/guides/migration-from-2-to-3.html#lifecycle-methods